site/app/controllers/EmailStatusController.php

```diff
@@ -33,4 +31,19 @@ public function getEmailStatusPage() {
             )
         );
     }
+
+    /**
+     * @Route("/superuser/email_status")
+     * @AccessControl(level="SUPERUSER")
+     * @return WebResponse
+     */
+    public function getSuperuserEmailStatusPage(): WebResponse {
+        $results = [];
+        $emailStatuses = $this->core->getQueries()->getAllEmailStatuses();
+        return new WebResponse(
+            EmailStatusView::class,
+            'showSuperuserEmailStatus',
+            $emailStatuses
+        );
+    }
```

Comment:
Let's move this function into `site/app/controllers/superuser/SuperuserEmailController.php`?

Reply:
Originally I intended `SuperuserEmailController.php` to be used for the email sending page while the `EmailStatusController.php` to be used for the email status page. Should I still move the function into `SuperuserEmailController.php`?